mava/utils/debugging/environments/switch_game.py

```diff
@@ -0,0 +1,154 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, Dict, List, Tuple, Union
+
+import gym
+import numpy as np
+from gym import spaces
+
+"""
+DIAL switch game implementation
+Adapted from https://arxiv.org/pdf/1605.06676.pdf.
+
+Actions:
+0 - Nothing
+1 - Tell
+"""
+
+
+class MultiAgentSwitchGame(gym.Env):
```

Comment:
Was this re-added on purpose?

Reply:
Good catch @KaleabTessera.

Reply:
Let me update the branch. This shouldn't be there.

Reply:
Thanks @KaleabTessera. This has been removed. I'm not sure how that got back in.